build.gradle

```diff
@@ -16,16 +16,23 @@ app {
     devSourcePaths += file("$rootDir/../lib-admin-ui/src/main/resources")
 }
 
+configurations {
+    devResources {
+        attributes {
+            attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, objects.named(LibraryElements, 'dev-resources'))
+        }
+    }
+}
+
 dependencies {
     include "com.enonic.xp:lib-content:${xpVersion}"
     include "com.enonic.xp:lib-context:${xpVersion}"
     include "com.enonic.xp:lib-portal:${xpVersion}"
     include "com.enonic.xp:lib-event:${xpVersion}"
-    include "com.enonic.lib:lib-admin-ui:${libAdminUiVersion}"
-    include "com.enonic.lib:lib-thymeleaf:2.0.0"
-    include "com.enonic.lib:lib-http-client:2.2.0"
+    include "com.enonic.lib:lib-thymeleaf:2.0.1"
+    include "com.enonic.lib:lib-http-client:2.2.1"
     include "com.enonic.lib:lib-cron:1.1.1"
-
+    devResources "com.enonic.lib:lib-admin-ui:${libAdminUiVersion}"
```

Comment:
you probably need to include `lib-admin-ui` as well, if you need its resources in jar.